.netlify/functions-internal/chunks/pages/keystatic-api_2116ae0c.mjs

```
@@ -1,23 +1,33 @@
-    ..._config,
-    clientId: (_config$clientId = _config.clientId) !== null && _config$clientId !== void 0 ? _config$clientId : tryOrUndefined(() => ({}).KEYSTATIC_GITHUB_CLIENT_ID),
-    clientSecret: (_config$clientSecret = _config.clientSecret) !== null && _config$clientSecret !== void 0 ? _config$clientSecret : tryOrUndefined(() => ({}).KEYSTATIC_GITHUB_CLIENT_SECRET),
-    secret: (_config$secret = _config.secret) !== null && _config$secret !== void 0 ? _config$secret : tryOrUndefined(() => ({}).KEYSTATIC_SECRET)
-  }, {
-    slugEnvName: 'PUBLIC_KEYSTATIC_GITHUB_APP_SLUG'
-  });
+  const handler = makeGenericAPIRouteHandler(
+    {
+      ..._config,
+      clientId:
+        (_config$clientId = _config.clientId) !== null &&
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)